api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -961,54 +961,40 @@ public DbUser getByUsernameOrThrow(String username) {
 
   @Override
   public void updateBypassTime(long userDatabaseId, AccessBypassRequest accessBypassRequest) {
-    final DbUser user =
-        getByDatabaseId(userDatabaseId)
-            .orElseThrow(
-                () ->
-                    new NotFoundException(
-                        String.format("User with database ID %d not found", userDatabaseId)));
+    getByDatabaseId(userDatabaseId)
+        .orElseThrow(
+            () ->
+                new NotFoundException(
+                    String.format("User with database ID %d not found", userDatabaseId)));
 
     final Timestamp newBypassTime;
-    final Timestamp previousBypassTime;
 
     final Boolean isBypassed = accessBypassRequest.getIsBypassed();
     if (isBypassed) {
       newBypassTime = new Timestamp(clock.instant().toEpochMilli());
     } else {
       newBypassTime = null;
     }
+
     switch (accessBypassRequest.getModuleName()) {
       case DATA_USE_AGREEMENT:
-        previousBypassTime = user.getDataUseAgreementBypassTime();
         setDataUseAgreementBypassTime(userDatabaseId, newBypassTime);
         break;
       case COMPLIANCE_TRAINING:
-        previousBypassTime = user.getComplianceTrainingBypassTime();
         setComplianceTrainingBypassTime(userDatabaseId, newBypassTime);
         break;
       case BETA_ACCESS:
-        previousBypassTime = user.getBetaAccessBypassTime();
         setBetaAccessBypassTime(userDatabaseId, newBypassTime);
         break;
       case ERA_COMMONS:
-        previousBypassTime = user.getEraCommonsBypassTime();
         setEraCommonsBypassTime(userDatabaseId, newBypassTime);
         break;
       case TWO_FACTOR_AUTH:
-        previousBypassTime = user.getTwoFactorAuthBypassTime();
         setTwoFactorAuthBypassTime(userDatabaseId, newBypassTime);
         break;
       default:
         throw new BadRequestException(
             "There is no access module named: " + accessBypassRequest.getModuleName().toString());
     }
-    logAdminUserAction(
```

Comment:
logged in UserServiceImpl.setBypasstimeWithRetries() which is called by each of the setXYZBypassTime()